python-sdk/dev/Dockerfile

```diff
@@ -11,7 +11,8 @@ RUN apt-get install -y --no-install-recommends \
         freetds-dev \
         libssl-dev \
         libkrb5-dev \
-        libmariadb-dev
+        libmariadb-dev \
+        pkg-config
```

Comment:
Why are we introducing this dependency?

Reply:
I couldn't build a docker without this package, it seems to be a dependency of mariadb: https://stackoverflow.com/questions/76533384/docker-alpine-build-fails-on-mysqlclient-installation-with-error-exception-can